# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ploticus/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For some reason strchr became a macro on my system lately and the protype is
wrong anyway ...

  - Rene Rebe <rene@exactcode.de>

--- pl231src/src/functions.c.vanilla	2005-03-06 12:16:41.160740088 +0100
+++ pl231src/src/functions.c	2005-03-06 12:16:57.089318576 +0100
@@ -636,7 +636,6 @@
 
 	if( hash == 1708 ) { /* $htmldisplay( var1, .. varN ) - convert embedded HTML problem chars (<,>,&) 
 			      * to their html esc sequences for all variables given */
-		char *strchr();
 		for( i = 0; i < nargs; i++ ) {
         		stat = TDH_getvalue( tok, arg[i], TDH_dat, TDH_recid );
 			if( strchr( tok, '&' ) != NULL ) GL_substitute( "&", "&amp;", tok );
